pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -26,15 +27,24 @@ import (
 
 // InitializeConnTrackDumper initializes the ConnTrackDumper interface for different OS and datapath types.
 func InitializeConnTrackDumper(nodeConfig *config.NodeConfig, serviceCIDRv4 *net.IPNet, serviceCIDRv6 *net.IPNet, ovsDatapathType ovsconfig.OVSDatapathType, isAntreaProxyEnabled bool) ConnTrackDumper {
+	var svcCIDRv4, svcCIDRv6 netip.Prefix
```

Comment:
Do you think it is better to change svcCIDRv4/v6 in agent.go?

Reply:
There is no need to. While I think that in the long term, we could replace all occurrences of `net.IP` with `netip.Addr` (and `net.IPNet` with `netip.Prefix`), it's better to do it incrementally with small PRs, starting with places providing the bigger benefits (in terms of performance). The configuration CIDRs for the Agent (podCIDR, svcCIDR) are individual values and don't account for any memory usage (while the FlowExporter connection store can potentially holds 1000s of IP addresses).